### PR TITLE
Various website fixes

### PR DESCRIPTION
**Pull request by marceloyuela**
Some style fixes and the addition of getProperty to the main controller (depreciating getParams).

### CODE REVIEW

File: Tools/GenerateWebsiteData.ps1 [hunk 1]
```diff
@@ -22,7 +22,8 @@ function convertToString ($obj)
 }
 
 # Get ParameterSets creates the parameter set hashtable for each cmdlet.
-# It takes in the get-help object for the cmdlet.
+# It takes in the System.Management.Automation.PSCustomObject object
+# for the cmdlet, which is acquird with Get-Help..
```
Comment (chrsmith) [inline Tools/GenerateWebsiteData.ps1]: "acquired via Get-Help".

- acquired
- One period at the end.

File: website_v2/gcloud-powershell-website-v2/www/static/app/main_controller.js [hunk 2]
```diff
@@ -46,14 +46,13 @@ app.controller('MainController',
...
-      };
-      
       /* Tells us if the object passed in is empty */
       this.isEmpty = function(linkObject) {
-        return (Object.keys(linkObject) === 0);
-      }
+        return (Object.keys(linkObject).length === 0);
+      };
+
+      /* Gets the relevant cmdlet property from the productInfo */
```
Comment (chrsmith) [inline website_v2/gcloud-powershell-website-v2/www/static/app/main_controller.js]: End the sentence with a period.

File: website_v2/gcloud-powershell-website-v2/www/static/app/main_controller.js [hunk 3]
```diff
@@ -46,14 +46,13 @@ app.controller('MainController',
...
+      this.getProperty = function(productInfo,  property) {
+        return productInfo[$routeParams.product][$routeParams.cmdlet][property];
```
Comment (chrsmith) [inline website_v2/gcloud-powershell-website-v2/www/static/app/main_controller.js]: Is this ever going to be called with something other than `mainCtrl.productInfo`? If not, consider just having this function take `property` and return `$scope.productInfo[...`.

**Comment by chrsmith** [comment]
Couple of suggestions, but LGTM.

**Comment by marceloyuela** [comment]
Should have fixed those issues.

**Pull request merged**